### PR TITLE
Make time period configurable

### PR DESCRIPTION
**Pull request by tomrf1**
We currently get Ophan's default of 3 mins, which is probably too short. This change will make it very easy to change.

### CODE REVIEW

File: src/main/scala/com/gu/contentapi/mostviewedvideo/OphanStore.scala [hunk 1]
```diff
@@ -32,6 +32,9 @@ trait OphanStore extends Http {
       left(CustomError(s"Failed to send request to Ophan for $url, response was ${response.code}"))
   }
 
+  private[this] def buildUrl(path: String, edition: Option[String], config: Config) =
+    s"${config.ophanHost}$path?${buildCountryQuery(edition)}mins=${config.ophanMinutes}&apiKey=${config.ophanKey}"
```
Comment (cb372) [inline src/main/scala/com/gu/contentapi/mostviewedvideo/OphanStore.scala]: Just to be consistent, I'd take the `&` out of `buildCountryQuery` and put it in here.

**Comment by cb372** [comment]
Looks good. But 3 minutes sounds quite short? For the most-viewed content that Shuttlerun collects, I have a feeling the time range is about 20 minutes.

**Comment by LATaylor-guardian** [comment]
I can't remember specifically, but I thought it was around 3 hours.

On Wednesday, April 6, 2016, Chris Birchall <notifications@github.com>
wrote:

> Looks good. But 3 minutes sounds quite short? For the most-viewed content
> that Shuttlerun collects, I have a feeling the time range is about 20
> minutes.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/guardian/most-viewed-video-uploader/pull/2#issuecomment-206473801>
>


-- 

*Luke Taylor*

Software developer, Content API

Guardian News & Media

-----

*luke.taylor@theguardian.com <luke.taylor@theguardian.com>*

-----

@luke_tay_taylor <https://twitter.com/luke_tay_taylor/>

-----

Kings Place, 90 York Way,

London N1 9GU

theguardian.com

​-----
​​Download the Guardian app for Android
<https://play.google.com/store/apps/details?id=com.guardian&hl=en_GB> and
iOS​ <https://itunes.apple.com/gb/app/the-guardian/id409128287?mt=8>

-- 

------------------------------
This e-mail and all attachments are confidential and may also be 
privileged. If you are not the named recipient, please notify the sender 
and delete the e-mail and all attachments immediately. Do not disclose the 
contents to another person. You may not use the information for any 
purpose, or store, or copy, it in any way.  Guardian News & Media Limited 
is not liable for any computer viruses or other material transmitted with 
or as part of this e-mail. You should employ virus checking software.
 
Guardian News & Media Limited is a member of Guardian Media Group plc. Registered 
Office: PO Box 68164, Kings Place, 90 York Way, London, N1P 2AP.  Registered 
in England Number 908396

File: src/main/scala/com/gu/contentapi/mostviewedvideo/OphanStore.scala (hunk 1 above)
Comment (tomrf1) [inline src/main/scala/com/gu/contentapi/mostviewedvideo/OphanStore.scala]: the edition is optional, so we may not want that `&`

**Comment by tomrf1** [comment]
@cb372 yes it's probably too short, I've made it clearer in the description that this was the intention. I've asked the relevant people if they have a preferred time range.

**Pull request merged**